python_modules/libraries/dagster-embedded-elt/dagster_embedded_elt/sling/resources.py

```diff
@@ -360,14 +365,14 @@ def replicate(
         self,
         *,
         replication_config: SlingReplicationParam,
-        dagster_sling_translator: DagsterSlingTranslator,
```

Comment:
This is a breaking change, I wonder if we can handle this gracefully with a deprecated param. We can remove it from the args, but if someone still specifies it as a keyword argument we should raise a deprecation warning. 

See here for example: https://github.com/dagster-io/dagster/blob/ce6c9a2f2f65bfef9b5ae47b69cc239181489658/python_modules/libraries/dagster-pagerduty/dagster_pagerduty/hooks.py#L22